src/engine/GameWorker.ts

```diff
@@ -264,18 +276,24 @@ async function onInit({
 
   const network: NetworkState = {
     hosting: false,
-    messages: [],
-    entityIdMap: new Map<number, number>(),
+    incoming: [],
+    networkIdToEntityId: new Map<number, number>(),
+    peerIdToEntityId: new Map(),
     peerId: "",
     peers: [],
     newPeers: [],
-    peerIdMap: new Map(),
+    peerIdToIndex: new Map(),
+    indexToPeerId: new Map(),
     peerIdCount: 0,
     localIdCount: 0,
     removedLocalIds: [],
     messageHandlers: {},
   };
 
+  const audio = {
+    tripleBuffer: audioTripleBuffer,
```

Comment:
We may want to use this for sharing other bits of data from GameWorker -> MainThread. Perhaps this should be in a non-audio specific section of the GameWorker state?